### ISSUE TITLE
compared with ShuffleNet V2

### ISSUE THREAD

**Issue by cs-heibao** [sacmehta/ESPNetv2#29]
@sacmehta 
hi, after read your paper and run the code, I've tried compared it with shufflenet v2 1×(implemented in tensorflow) for image classification, their GPU(1080Ti) Speed as follow:
shufflenet v2 1×: 3ms/per image
ESPNet V2: 13ms/per image
I want to make sure my test process is right? (or the actual forward infer is the above speed), thanks

**Comment by sacmehta** [comment]
ESPNetv2 which version? ESPNetv2-1.0?

**Comment by sacmehta** [comment]
ESPNetv2-2.0 full 32-bit precision network  (not quantized one) image classification on iPhone7 is 100+ FPS, which means it is running at close to 10ms per image. So, you should get better speed on your GPU.

Also, it is not good to compare PyTorch and TensorFlow models directly.

**Comment by cs-heibao** [comment]
@sacmehta 
how about the PyTorch model speed on 1080Ti graphics card?
